### ISSUE TITLE
Add Aseprite file support

### ISSUE THREAD

**Issue by NOTMEE12** [pygame-community/pygame-ce#2184]
Make so `pygame.image.load` supports loading files that have `.aseprite` or `.ase` as extension.
For now I have to create two files:
- one ending with extension that pygame-ce supports
- second one that is ending with `.aseprite` so it supports layers, etc.

with larger projects, *"doubling"* files just to have the option to edit them, can take space.

**Comment by FirTreeMan** [comment]
That would entail building the image from the file with layers, resulting in significant overhead. Simply exporting a `.png` file and taking up a little space seems to me to be the best scenario. 
Additionally, there are many different file types for different editing softwares; is it also expected to support all of those?
In concept it would be nice to be able to directly use the file after saving it, but in execution it would be a lot of work for very little benefit, sacrificing runtime to save a few KB or MB of storage space. I really don't think that small amount of space is worth all the overhead and work to implement this feature.

**Comment by Starbuck5** [comment]
As FirTreeMan says, pygame-ce doesn't have support for other editing software file formats (paint.net, gimp, photoshop), and it's pretty reasonable to need to export your files to a generic file format.

I find space is a concern more when distributing than when in development. So to address the space concern I'd say don't ship them in the distributable game.

I did actually look around to see if there was a quality project that we could just slot in for this support, or to point you towards to use with pygame-ce. I didn't really find much, and what I did find seemed to load the files as lists of attributes, not as raster graphics (pixels) that could be loaded into pygame-ce.

If you're up for an adventure, making an asesprite rasterizing Python library could be fun. Make it generic enough and it could work with PIL, pygame, etc.
